.gitignore

```diff
@@ -2,6 +2,4 @@
 *.iml
 .gradle/
 .idea/
-app/app.iml
-imagecrop/imagecrop.iml
```

Comment:
I remove them because it has no effect in my cli... I still have them in the "not staged".